library/chacha20.c

```diff
@@ -210,6 +209,22 @@ int mbedtls_chacha20_starts(mbedtls_chacha20_context *ctx,
     return 0;
 }
 
+int mbedtls_chacha20_set_iv(mbedtls_chacha20_context *ctx,
```

Comment:
Having seen this workaround I'm thinking that we maybe should avoid touching/adding new Mbed TLS 'classic' APIs. The PSA Crypto implementation could do this same thing internally. The updated API docs in `mbedtls_chacha20_setkey` feel a bit sketchy to me, since existing apps are not going to be doing that, and the text makes it sound like you _must_ do it. What do you think?